src/components/Layout/LeftSidebar.tsx

```diff
@@ -130,6 +130,26 @@ const constructProductNavData = (
     return {
       name: product.name,
       icon: activePageTree[0]?.page.name === product.name ? product.icon.open : product.icon.closed,
+      onClick: () => {
```

Comment:
When a top-level accordion item is clicked, then it tries to bring the header row into view if it isn't visible - this happens after the opening animation has concluded, to ensure that the `scrollTo` has the right point of reference.